_posts/2015-12-09-foreman-community-newsletter-december-2015.md

```diff
@@ -0,0 +1,81 @@
+    For more information please read the [announcement](https://groups.google.com/forum/#!topic/foreman-users/YQNfxUzG_As).
+-   The development infrastructure has seen some updates - Redmine has been updated to version 2.6.8 and Jenkins has been updated to version 1.625.2.
+
+### New media and blogs
+
+-   The latest [community demo](https://www.youtube.com/watch?v=RCqwTtIXRSw) took place on November 19th. Highlights:
+    - yum repo export/import (beav) - https://www.youtu.be/RCqwTtIXRSw
+    - remote execution iteration 4 (stbenjam) - https://www.youtu.be/RCqwTtIXRSw?t=502
+    - foreman_ansible updates (dlobatog) - https://www.youtu.be/RCqwTtIXRSw?t=743
+    - scoped search for content hosts/subscriptions (jomitsch) - https://www.youtu.be/RCqwTtIXRSw?t=1501
+    - Rails 4 migration (the_doctor) - https://www.youtu.be/RCqwTtIXRSw?t=1689
+    - compliance reports for foreman_openscap (oprazak) - https://www.youtu.be/RCqwTtIXRSw?t=2201
+    - community announcements (gwmngilfen) - https://www.youtu.be/RCqwTtIXRSw?t=2359
+-   We held a deep dive about [Windows Imaging](https://www.youtube.com/watch?v=_R8-0KAkoPc) with Daniel Helgenberger. Thanks Daniel!
+-   We continuted our user case study series with Chris "discr33t" Pisano talking about his [High Availability](https://www.youtube.com/watch?v=bJ5a67SSA-s) setup with Foreman. Thanks Chris!
+-   Lukas Zapletal (lzap) gave a talk about Foreman at the Pilsen University. Read his report [here](https://groups.google.com/forum/#!topic/foreman-dev/l3-4a6pQTZU). Thanks Lukas!
+-   Daniel Lobato Garcia (dlobatog) gave a talk about [Rails security](https://speakerdeck.com/elobato/20-actionable-tips-to-secure-your-rails-application-rails-israel-15) at the Rails Israel conference. He also joined the Israeli Foreman team members to show off the project at the Red Hat booth.
+
+### Upcoming events
+
+-   This Thursday (Dec. 10th) is community demo time again.
+    Check out the [Latest Demo Information](http://projects.theforeman.org/projects/foreman/wiki/Current_Sprint_Information) page for agenda and viewing links.
+-   The CFP for [FOSDEM](https://fosdem.org/) and [Config Management Camp](http://cfgmgmtcamp.eu/) has closed.
+    We are awaiting the agenda to be announced, hopefully with many Foreman related talks.
+    Lookin forward to seeing you all in Belgium on January 30 - February 2 , 2016.
+    There has been a proposal to have an extra Foreman Hackday on February 3rd, right after Config Management camp.
+    If you are interested, please fill out this extra short [survey](https://docs.google.com/forms/d/1LI5-L7sgWoZqFvph_hUgkNOEcH0fZ5pRyrGYFqS6DDs/viewform).
+
+You can follow us on Twitter: [@ForemanProject](https://twitter.com/ForemanProject).
+If you tweet about foreman, please use the hashtag [\#theforeman](https://twitter.com/hashtag/theforeman).
+
+We also have a [Facebook group](https://www.facebook.com/groups/theforeman/) which you can join for updates & discussion.
```

Comment:
Social media info is automatically added in a footer of the blog post layout, no need for you to add it :)